app/controllers/participants_controller.rb

```diff
@@ -2,9 +2,11 @@ class ParticipantsController < ApplicationController
   before_action :set_participant, only: [:show]
 
   def show
+    # TODO CHECK IF CORRECT
```

Comment:
Reminder to delete comment before merge. Works for me (as in I can arrive at /participants/:id pages in staff but not participant)